lib/synced/strategies/updated_since.rb

```diff
@@ -44,7 +44,11 @@ def initial_sync_since
 
       def updated_since
         instrument("updated_since.synced") do
-          [@timestamp_strategy.last_synced_at, initial_sync_since].compact.max
+          last_synced_at_offset = ENV.fetch("LAST_SYNCED_AT_OFFSET", 0).to_i
+          [
+            @timestamp_strategy.last_synced_at&.advance(seconds: last_synced_at_offset),
```

Comment:
advance suggest moving ahead of time, but this use case seems to go back always. maybe better to use `ago`?

Reply:
I wanted to be more generic - if offset is positive - move forward (although don't see why would anybody need to move it forward), if negative - move backwards.
For `ago` the ENV variable should be named differently, so it won't be confusing about using positive or negative numbers. But quite frankly I have no idea how to name that.

Reply:
you could always extract this to 
```
[last_synced_at_with_offset, initial_sync_since].compact.max

def last_synced_at_with_offset
  return if @timestamp_strategy.last_synced_at.blank?
  @timestamp_strategy.last_synced_at + last_synced_at_offset
end
```

Reply:
You're using the `&` but is there a spec for it?